features/accounts/edit/profile-form/profile-form.tsx

```diff
@@ -59,6 +60,15 @@ export const ProfileForm: FC = () => {
     // push(value);
   };
 
+  console.log(control);
+  const handleInputChange = event => {
+    const inputValue = event.target.value;
+    // Разрешаем вводить только цифры и специальные символы (как запятая, точка и т.д.)
+    const filteredValue = inputValue.replace(/[^\d,-./]/g, '');
+
+    setValue(filteredValue);
```

Comment:
It looks like this piece of code is garbage left over from a failed experiment. Don't leave garbage like this, it will lead to bugs if the reviewer misses this code. Consider making an experimental topic-branch instead, which will not be posted to the remote repository.